### ISSUE TITLE
[Request] Custom EMC documentation

### ISSUE THREAD

**Issue by Cobra1117** [pahimar/Equivalent-Exchange-3#873]
It would be helpful to have a very brief write-up on how to add custom EMC values, including tips on how to determine what value to assign an item. (For example, ores seem to be always given the same EMC value as their smelted form.) I know that Pahimar added EMC values for many of the ores/items on Forgecraft, so some notes on his thought process would be helpful to us server admins.

**Comment by reapersremorse** [comment]
You can go through the cofh config to see where certain ore spawns and
determin rarity of the ore, once you know the rarity from between stone and
diamond you should be able to figure out the emc value.

**Comment by Cobra1117** [comment]
Oooh. Nice suggestion! Many thanks!

(Although I should point out that this is exactly the kind of tip that should be documented in a wiki somewhere!) :-)

**Comment by Cobra1117** [comment]
Hrm... I must be misreading something (or perhaps I'm over-thinking this). The numbers here don't seem to really work out. For example:

Coal generates up to 20 clusters of 16 ore. Each ore generates 1 coal, with up to 3 "bonus" coal from Fortune 3. Since coal has an EMC of 32, that gives you the following average EMC values per chunk:
No fortune: 10,240 EMC
Fortune giving 1/2 max bonus: 25,600 EMC
Fortune giving max bonus: 40,960

However, if we look at Diamonds (cluster size 7, 1 cluster per chunk, up to 3 "bonus" diamonds from fortune, EMC value of 8,192 each), we get these average EMC values per chunk:
No fortune: 57,344 EMC
Fortune giving 1/2 max bonus: 143,360 EMC
Fortune giving max bonus: 229,376 EMC

And other ores have similarly widely-varying numbers. Is this intentional, or am I missing an element in this equation?

**Comment by reapersremorse** [comment]
Fortune is not factored and will not be factored for dyn emc. There are
many exploits built into the game

Think of it like this. How long would it take you to strip mine a 3x3 chunk
area at diamond level?? How uvh stone, iron, gold, coal and diamonds would
you find???

The emc values dont need changed, but in the cofh config, you can change
the ore spawn

**Comment by Cobra1117** [comment]
That makes sense. How do you recommend handling pricing of ores that produce multiple items? (Redstone and Forestry's Apatite, for example)

**Comment by reapersremorse** [comment]
I wouldnt know about apatite but if its set by dyn emc,  Its your game so
im sure you would figure what emc values u want.

I quarried a 3x3 chunk area from max to bedrock. I used the ammount of each
item and blocks in my ae system to figure out what emc value everything
should be set to.

(This was done in a extreme hills + m biome so emeralds would show)

Copper and such were around the same as iron etc etc.

My emc values are as standard as possible but thats derivative to ore spawn
in my game which is tweaked.

I wouldnt make ore researchable and having an emc value is useless unless
you have no ore doubleing mechanic or your in need of quick emc, but if u
want the emc for the ore then just set the ores emc to the ammount of 1
item that drops from it, this would keep it off the op rout of play.

**Comment by GTMelo** [comment]
I've only started playing EMC values, but allow me to chime in. I usually compare to something from vanilla, and add or take some weight on the fly. For example, apatite comes in greater number, yelds more and sits around surface level, so I think it feels like cheaper than coal. Coal is 32 EMC (I think), so I'd gauge apatite to somewhere between 1 and 16. Less if I play around and still think it's giving me too much energy. Copper feels a bit cheaper than iron, and tin a little more. So I set it to 220 copper and 310 tin. Purely arbitrary numbers, but meh, kinda feels right.
